app/presenters/publishing_api/payload_builder/people.rb

```diff
@@ -0,0 +1,44 @@
+module PublishingApi
+  module PayloadBuilder
+    class People
+      attr_reader :item, :key
+
+      def self.for(item, key)
+        self.new(item, key).call
+      end
+
+      def initialize(item, key)
+        @item = item
+        @key = key
+      end
+
+      def call
+        people
+      end
+
+    private
+
+      def people
+        return {} unless people_present?
+
+        { key => people_from_role_appointments + person_from_role_appointment }
+      end
+
+      def people_present?
+        people_from_role_appointments.present? || person_from_role_appointment.present?
+      end
+
+      def people_from_role_appointments
```

Comment:
Could this be refactored to a single method something like:

```
(Array(ed.try(:role_appointments)) + Array(ed.try(:role_appointment)))
  .map(&:person)
  .map(&:content_id)
```

I'm not sure that's actually an improvement. Using `Array(item.try...` in each method might be though.